spec/controllers/media_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+describe MediaController, vcr: { record: :new_episodes } do
+  let(:video) { StacksMediaStream.new(id: 'bb582xs1304', file_name: 'bb582xs1304_sl', format: 'mp4') }
+
+  describe '#download' do
+    subject { get :download, id: 'bb582xs1304', file_name: 'bb582xs1304_sl', format: 'mp4' }
+
+    it 'sends the video url to the user' do
+      expect(controller).to receive(:send_file).with(video.path).and_call_original
+      subject
+    end
+
+    it 'loads the video' do
+      subject
+      video = controller.instance_variable_get(:@media)
```

Comment:
Is it possible to use `assigns` here instead of `instance_variable_get`?  If not, no worries, just a red-flag.